content/applications/websites/website/publish/translate.rst

```diff
@@ -1,39 +1,85 @@
-===========================
-How to translate my website
-===========================
+===================
+Website translation
+===================
 
-Overview
-========
+The contents of your website pages (i.e. text strings) can be translated into different languages
+directly on your website.
 
-In addition to creating great modern websites, Odoo gives you the
-possibility to translate it in different languages.
+.. note::
+   Your website is displayed in the language that matches the visitor's browser's language, unless
+   that particular language has not been installed. In this case, the website is displayed in the
+   :ref:`default language <default-language>`. The visitor can still select another language in
+   the language menu.
 
-Process
-=======
+Installing languages
+====================
 
-Once your website is created, you have the opportunity to translate it
-in as many different languages as you want.
+To translate your website, you first have to add the required language(s):
 
-You can only translate your website manually, follow the next step.
+#. Go to your website.
+#. Scroll at the bottom of the page to the **language menu**.
+#. Click the language and select :guilabel:`Add a language`.
 
-Now go to your website. On the bottom right corner of the page, click on
-**Add a language**.
+   .. image:: translate/website-add-language.png
+      :alt: Add a language to your website.
 
-.. image:: translate/translate_website01.png
-    :align: center
+#. Click the :guilabel:`Languages` field and select the required language from the drop-down
+   list. Repeat this step for each additional language.
+#. Click the :guilabel:`Add` button.
 
-Choose the language in which you want to translate your website and then
-click on **Load.**
+.. tip::
+   - You can also edit your website's languages from the backend, in the :guilabel:`Settings`. Go to
+     :menuselection:`Website –> Configuration –> Settings`, and add/remove the required
+     languages in the :guilabel:`Languages` field.
 
-.. image:: translate/translate_website02.png
-    :align: center
+   .. _default-language:
 
-You will see that Now, next to English there is also French, which means
-that the page for the translation has been created. You can also see
-that some of the text has been translated automatically.
+   - The website's default language can be defined in the :guilabel:`Default` field in the
+     :guilabel:`Settings`.
 
-.. image:: translate/translate_website03.png
-    :align: center
+Translating the contents
+========================
 
-To translate the content of the website, click on **Translate** (here
-**Traduire** since we want to translate the website in French).
+Once the required languages have been added, you can translate the contents of your website. To do
+so, go to your website, select the required language from the language menu and click the
```

Comment:
This is very nitpicky but "required language" sounds a bit like it's a language required by the system. Perhaps "once a new language has been added, you can translate the contents of your website (...). Select the language from the menu and (...).